stdlib/typing.pyi

```diff
@@ -99,7 +100,7 @@ _T_contra = TypeVar("_T_contra", contravariant=True)  # Ditto contravariant.
 _TC = TypeVar("_TC", bound=Type[object])
 
 def no_type_check(arg: _F) -> _F: ...
-def no_type_check_decorator(decorator: _F) -> _F: ...
+def no_type_check_decorator(decorator: Callable[_P, Any]) -> Callable[_P, Any]: ...  # type: ignore
```

Comment:
Why not use a `TypeVar` for the return type?

Reply:
I was not sure, if I should enforce it, because the one who implements its own `no_type_check` decorator could change the return type, if he wants to.

Reply:
The implementation is, ignoring `no_type_check` because it returns the same type as its argument:

```python3
def no_type_check_decorator(decorator):
    @functools.wraps(decorator)
    def wrapped_decorator(*args, **kwds):
        return decorator(*args, **kwds)
    return wrapped_decorator
```

And we can simplify that further, since `wrapped_decorator` and `decorator` are now the same type:

```python3
def no_type_check_decorator(decorator):
    return decorator
```

So the `decorator` argument and the returned callable always have the same exact arguments, and the same exact return type. Am I missing something?